examples/powertools-examples-core-utilities/cdk/app/pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>software.amazon.lambda.examples</groupId>
-    <version>1.16.1</version>
+    <version>1.17.0</version>
```

Comment:
This is wrong (but it's also wrong in `v2`) . Should we fix it here?

Reply:
It appears to only be this example, which follows as it was added after we branched v2

Reply:
it does not work if I put 2.0.0-SNAPSHOT, because it cannot find the installed version (2.0.0-SNAPSHOT) of the libraries. I didn't dig into why, but probably because it is built in docker and there's no v2 in central...

Reply:
let's see that apart, so that we finish the merge